### ISSUE TITLE
Config Files Not Working.

### ISSUE THREAD

**Issue by M7ilan** [l02162010/VSCode-Project-Structure#4]
Hello, Thank you for your lovely extention.

I have an issue where when I use the command palette and click `Generate Project Structure: Filtered files` I got the `docs` folder containing an empty `.project_structure_filter` file and the project structure in `project_structure_filtered.txt` but I have the `.gitignore` with the files I don't want to see in the project structure like `node_modules` which makes the generating process take a bit of time, I also tried to add `node_modules` in `.project_structure_filter` located inside the docs and the root of the project but I still get the full structure.

Note that the I have enabled the `Use Git Ignore` setting.

**Comment by fabrizionastri** [comment]
First run the `Generate Project Structure: All files` command to generate the `docs` folder and an empty `.project_structure_ignore` file. Then add `node_modules` to the `.project_structure_ignore` file. Let me know if that works.

**Comment by M7ilan** [comment]
@fabrizionastri It works! here's what I did:
1. Run the `Generate Project Structure: All files` in the command pallete.
2. Now, I have a 'docs' folder and a `.project_structure_ignore` file. I use this ignore file to specify the files I want to ignore.
3. Run the `Generate Project Structure: Filtered files`
But I wonder.. is there a way to targeting one folder instead of ignoring? like all I want is to generate a structure for `app/` folder.

**Comment by fabrizionastri** [comment]
Yes, simply add `app/` to the `.project_structure_filter` file. Please read the extension documentation and let me know if there anything unclear or missing.

**Comment by M7ilan** [comment]
Since the `app` folder is located within my `src`, I wrote `src` in the `.project_structure_filter` file and `node_modules` in the ignore file. But, it seems the filter is not working properly, as it generates the entire project except for the `node_modules`.

The documentation seems clear, but the filter not working properly is confusing. I suggest making the process easier by adding a single command to create the project structure. This command would use `.project_structure_ignore` and `.project_structure_filter` files in the project's root, similar to `.gitignore` or `.prettierignore`. Or, we could use a config file like `.prettierrc.json` for settings.

I'll contribute as best as I can when I have time.

**Comment by fabrizionastri** [comment]
Can you please test this extension and see if you have the same problem : [VSCode-Project-Structure-Plus](https://marketplace.visualstudio.com/items?itemName=FabrizioNastri.vscode-project-structure-plus). This is the version in which I developed the new functionalities.

I have not tested the updated VSCode-Project-Structure extension after it integrated my changes.

https://github.com/fabrizionastri/VSCode-Project-Structure-Plus

**Comment by engine1c** [comment]
Boy, you are a funny person!
The structure is being created, but...
.project_structure_filter
  .project_structure_ignore
DOESN'T WORK AT ALL!
Well, you can somehow test what you are doing :)

**Comment by engine1c** [comment]
simple

.project_structure_filter
lib
assets


.project_structure_ignore
.git
.vscode
.dart_tool
build
android
linux
macos
web
windows

project_structure_filtered.txt
--- Folder Structure ---
[.dart_tool]
    ├── [dartpad]
        └── web_plugin_registrant.dart
    ├── package_config.json
    ├── package_config_subset
    └── version
[.git]
    ├── COMMIT_EDITMSG
    ├── config
    ├── description
    ├── FETCH_HEAD
    ├── HEAD
    ├── [hooks]
        ├── applypatch-msg.sample
        ├── commit-msg.sample
        ├── fsmonitor-watchman.sample
        ├── post-update.sample
        ├── pre-applypatch.sample
        ├── pre-commit.sample
        ├── pre-merge-commit.sample
        ├── pre-push.sample
        ├── pre-rebase.sample
        ├── pre-receive.sample
        ├── prepare-commit-msg.sample
        ├── push-to-checkout.sample
        └── update.sample
    ├── index
    ├── [info]
        └── exclude
    ├── [logs]
        ├── HEAD
        └── [refs]
            ├── [heads]
...................................... 
etc.
and the idea is EXCELLENT and NEEDED!
It might be possible to finish it.
Good luck!

**Comment by l02162010** [comment]
Dear friends, I apologize. I didn't realize there were so many bugs in the project. Recently, I have been preoccupied with work and didn't pay attention to this. I am very sorry. I will find time in the near future to fix these bugs and try to optimize the functions. Thank you for your usage and feedback

@fabrizionastri thank you again for help me reply the comments

**Comment by fabrizionastri** [comment]
> Can you please test this extension and see if you have the same problem : [VSCode-Project-Structure-Plus](https://marketplace.visualstudio.com/items?itemName=FabrizioNastri.vscode-project-structure-plus). This is the version in which I developed the new functionalities.
> 
> I have not tested the updated VSCode-Project-Structure extension after it integrated my changes.
> 
> https://github.com/fabrizionastri/VSCode-Project-Structure-Plus

Can you please test this extension and see if you have the same problem : [VSCode-Project-Structure-Plus](https://marketplace.visualstudio.com/items?itemName=FabrizioNastri.vscode-project-structure-plus). This is the version in which I developed the new functionalities.

I have not tested the updated VSCode-Project-Structure extension after it integrated my changes.

https://github.com/fabrizionastri/VSCode-Project-Structure-Plus

**Comment by anrybalka** [comment]
I realized that the extension can only read the last line in .gitignore and .project_structure_ignore.
That is, the extension can ignore only 2 folders. One folder that is indicated by the last line in .project_structure_ignore and the second folder that is indicated by the last line in .gitignore
